omas/machine_mappings/_efit.json

```diff
@@ -80,7 +80,7 @@
   "treename": "{EFIT_tree}"
  },
  "equilibrium.time_slice.:.boundary_separatrix.closest_wall_point.distance": {
-  "TDI": "data(\\{EFIT_tree}::TOP.RESULTS.AEQDSK.DSEP)/100.",
+  "TDI": "data(\\{EFIT_tree}::TOP.RESULTS.AEQDSK.DRSEP)/100.",
```

Comment:
This is a different variable.  If MDS+ isn't saving `DSEP` it should be computed from `min(GAPIN,GAPTOP,GAPOUT,GAPBOT)`